templates/web/pages/user_form/user_form_page.tt.html

```diff
@@ -60,44 +60,57 @@
+                            <label for="pro">
+                                <input type="hidden" name="[% field.field %]" value="[% field.value %]">
+                            </label>
+                        [% END %]
 
-            [% FOREACH field IN section.fields %]
-                [% IF field.field == 'userid' %]
-                    [% IF type == 'edit' %]
-                        <label for="[% field.field %]"> [% field.label %]</label>
-                        <p id="userid"><a href="/editor/[% field.value %]">[% field.value %]</a></p>
-                    [% ELSE %]
-                        <label for="[% field.field %]">  [% lang(field.label) %] [% lang('username_info')  %]</label>
-                        <input type="text" id="[% field.field %]" name="[% field.field %]" value="[% field.value %]" autocomplete="username" onkeyup="normalize_string_value(this)" />
                     [% END %]
-                [% ELSIF field.type == 'text' %]
-                        <div [% IF section.id == "professional" %] class="pro_org_display" [% END %]>
-                            <label for="[% field.field %]">[% field.label %]</label>
-                            <input type="text" id="[% field.field %]" name="[% field.field %]" value="[% field.value %]" [% IF section.id == "teams" %] onkeyup="normalize_string_value(this)" [% END %]/>
-                        </div>
-                [% ELSIF field.type == 'password' %]
-                    <label for="[% field.field %]">[% lang(field.label) %]</label>
-                    <input type="password" id="[% field.field %]" name="[% field.field %]" value="[% field.value %]" autocomplete="new-password"/>
-                [% ELSIF field.type == 'email' %]
-                    <label for="[% field.field %]">	[% field.label %]</label>
-                    <input type="email" id="[% field.field %]" name="[% field.field %]" value = "[% field.value %]" autocomplete="email"/>
-                [% ELSIF field.type == 'checkbox' %]
-                    [% IF accepted_organization && section.id == "professional" %]
-                        <p>[% f_lang('f_this_is_a_pro_account_for_org', {'org' => accepted_organization}) %]</p>
-                    [% ELSE %]
-                        <label for="[% field.field %]">
-                            <input type="checkbox" id="[% field.field %]" name="[% field.field %]" [% IF field.value == 'on' %]checked="checked"[% END %] />
-                            [% field.label %]
-                        </label>
-                    [% END %]
-                [% ELSIF field.type == 'hidden' %]
-                    <label for="pro">
-                        <input type="hidden" name="[% field.field %]" value="[% field.value %]">
-                    </label>
+                </div>
+                <!-- show video about account creation beside user form section-->
+                [% IF section.id != "professional" AND section.id != "contributor_settings" %]
```

Comment:
the condition should probably be reversed: instead of showing the video when the section is not something and not something else, it should be shown only for the section we want it to be shown. Otherwise it will be shown in other sections (e.g. the administrator section, or any new section we add later)